renpy/python.py

```diff
@@ -353,7 +353,19 @@ def reset_store_changes(name):
 # the rollback list before rollback or serialization.
 
 
-class NoRollback(object):
+class SlottedNoRollback(object):
+    """
+    :doc: slottednorollback class
+
+    Instances of classes inheriting from this class do not participate in
+    rollback. Objects reachable through an instance of a SlottedNoRollback
+    class only participate in rollback if they are reachable through other
+    paths.
+    """
+    __slots__ = tuple()
```

Comment:
The use of the `()` litteral is safer and faster than a call to the tuple constructor.

Reply:
Done